modules/bitgo/src/v2/index.ts

```diff
@@ -1,3 +1,17 @@
+/* eslint-disable import/no-internal-modules */
+/**
+ * @deprecated after version 14.0.0
+ * Retained for short term backwards compatibility - migrate to: @bitgo/sdk-api and @bitgo/sdk-core
+ */
+export * from '@bitgo/sdk-core/dist/src/bitgo/baseCoin/baseCoin';
```

Comment:
can we just `export * from '@bitgo/sdk-core` instead of digging into dist folders?

Reply:
Yeah, it's my conflict between specific backwards compatibility or "adding more features" by exporting everything. :/

Reply:
I have some overlapping types to resolve if doing it that way :(